.spacelift/testinfra/infra/main.tf

```diff
@@ -0,0 +1,24 @@
+module "spacelift" {
+  # Since we're using this internally for testing, let's not pin to a specific version.
+  source = "github.com/spacelift-io/terraform-aws-spacelift-selfhosted"
+
+  region          = var.aws_region
+  rds_engine_mode = "provisioned"
+  rds_instance_configuration = {
+    "primary" : {
+      instance_identifier : "primary"
+      instance_class : "db.serverless"
+    }
+  }
+
+  rds_serverlessv2_scaling_configuration = {
+    min_capacity = 0.5
+    max_capacity = 5.0
```

Comment:
Maybe let's drop the max capacity here since we don't actually expect anything to run on it?